core/src/main/com/ruslan/entity/book/Book.java

```diff
@@ -1,124 +1,52 @@
 package com.ruslan.entity.book;
 
+
+import com.ruslan.entity.BaseEntity;
+import com.ruslan.entity.order.Order;
+import com.ruslan.entity.request.Request;
+import jakarta.persistence.*;
+import lombok.*;
+
 import java.io.Serializable;
 import java.time.LocalDate;
-import java.util.Objects;
+import java.util.List;
+
+@Entity
+@Table(name = "book", schema = "bookstore")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString(exclude = {"listOrder", "listRequests"}, callSuper = true)
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class Book extends BaseEntity<Integer> implements Serializable {
 
-public class Book implements Serializable {
-    private int id;
-    private BookStatus status;
     private String title;
     private String author;
-    private int price;
-    private LocalDate datePublication;
-
+    private Integer price;
+    @Enumerated(EnumType.STRING)
+    @Column(columnDefinition = "ENUM('IN_STOCK', 'NOT_AVAILABLE', 'OUT_OF_STOCK')")
```

Comment:
you don't need to set specific values because it will be saved as a string value in DB

Reply:
I added this annotation coz I had row in DDL `status` enum('IN_STOCK', 'NOT_AVAILABLE', 'OUT_OF_STOCK') NOT NULL.
I thought that setting the enum types for the column when creating the table was a good idea. 
I changed type column to VARCHAR(255) and removed this annotation.